src/storage/page_buffer.c

```diff
@@ -10397,6 +10397,7 @@ pgbuf_is_consistent (const PGBUF_BCB * bufptr, int likely_bad_after_fixcnt)
 	  return consistent;
 	}
 
+      perfmon_inc_stat_to_global (PSTAT_PB_NUM_IOREADS);
```

Comment:
since this is a debugging function, to collect stats is not so required.
If you think the new interface is useful (for future), we can have it.